src/components/NumberBadge/NumberBadge.stories.tsx

```diff
@@ -15,20 +16,37 @@ const Template: Story<NumberBadgeProps> = args => <NumberBadge {...args} />
 export const Basic = Template.bind({})
 
 Basic.args = {
-  children: 8,
+  children: 80,
 }
 
-const TONES: NumberBadgeTone[] = [`NEUTRAL`, `WARNING`, `DANGER`]
+const TONES: NumberBadgeTone[] = [`NEUTRAL`, `WARNING`, `DANGER`, `SUCCESS`]
 
 export const Tones = () =>
   TONES.map(tone => (
     <StoryPropVariant propName="tone" propValue={tone}>
-      <NumberBadge key={tone} tone={tone}>
-        8
+      <NumberBadge key={tone} tone={tone} withBorder={true}>
+        80
       </NumberBadge>
     </StoryPropVariant>
   ))
 
 Tones.story = {
   decorators: [withVariationsContainer],
 }
+
+export const withBorder = () => (
```

Comment:
```suggestion
export const WithBorder = () => (
```
To follow the PascalCase convention of CSF (https://storybook.js.org/docs/react/api/csf#named-story-exports)